### PR TITLE
add feature-engine

### PR DESCRIPTION
**Pull request by solegalli**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/feature_engine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```
Comment (scopatz) [inline recipes/feature_engine/meta.yaml:22]: Does this not depend on scikit-learn?

Comment (solegalli) [reply recipes/feature_engine/meta.yaml:22]: it does. Scikit-learn, pandas, numpy, scipy and statsmodels, as per:
https://github.com/solegalli/feature_engine/blob/master/requirements.txt

do I need to include all those in the meta.yaml?

Comment (solegalli) [reply recipes/feature_engine/meta.yaml:22]: @scopatz do I need to make the packages part of the meta?

**Comment by solegalli** [comment]
@scopatz any idea why this is failing? what else do I need to do? thank you!

**Comment by scopatz** [comment]
It is failing because it can't download the source tarball: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=193168&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=543

Do you have the URL correct?

**Comment by solegalli** [comment]
@scopatz I updated the url, it should work now, or at least I can download it manually from here: https://pypi.io/packages/source/f/feature_engine/feature_engine-0.5.15.tar.gz
But it still fails, could you tell me why? can I check somehow what the error is?

**Comment by solegalli** [comment]
@scopatz sorry to bother again, errors because it can't read the VERSION. Why could that be?

**Comment by dopplershift** [comment]
Are you sure `VERSION` is actually in the tarball? A lot of packages run into problems where the `LICENSE` file isn't in the tarball because it's not included by default and isn't added in `MANIFEST.in`.

**Comment by solegalli** [comment]
bravo! I finally nailed it lol. Thank you guys for your support @scopatz @dopplershift

**Pull request merged**